src/lib/kwlib.sh

```diff
@@ -695,6 +695,25 @@ function get_current_env_name()
   return "$ret"
 }
 
+# This function returns "${current_env}-base64(${PWD})".
+#
+# @env_name: The name of the environment to encode
+#
+# Return:
+# Returns 0 and prints ${current_env}-base64(${PWD}) if env_name is provided.
+# Returns 1 if env_name is not provided.
+get_env_name_encoded_with_pwd() {
```

Comment:
This is not the codestyle adopted by kw, please, read the below page and review your entire PR.

 https://kworkflow.org/content/codingstyle.html